OpenRA.Mods.Common/Traits/Parachutable.cs

```diff
@@ -71,7 +72,8 @@ public void OnLanded()
 			if (sequence != null && palette != null)
 				self.World.AddFrameEndTask(w => w.Add(new Explosion(w, self.OccupiesSpace.CenterPosition, sequence, palette)));
 
-			self.Kill(self);
+			if (self.World.ActorMap.GetActorsAt(self.Location).Any(a => a != ignore))
```

Comment:
What's the point of this?

You have blown the parachuted actor up by this point - even playing it's explosion.